<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shrishti_BT" failures="1" tests="3" name="CheckoutTest" time="31.363" errors="0" timestamp="2023-06-13T19:25:10 IST">
  <testcase classname="ECommerce.tests.CheckoutTest" name="EcommerceE2E" time="7.052"/>
  <testcase classname="ECommerce.tests.CheckoutTest" name="EcommerceE2E" time="6.743"/>
  <testcase classname="ECommerce.tests.CheckoutTest" name="OrderHistory" time="3.073">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ECommerce.tests.CheckoutTest.OrderHistory(CheckoutTest.java:92)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- OrderHistory -->
</testsuite> <!-- CheckoutTest -->
